### PR TITLE
[JENKINS-39542] Test results trimming null string fix.

### PR DESCRIPTION
**Pull request by imeredith**
# Description

See [JENKINS-39542](https://issues.jenkins-ci.org/browse/JENKINS-39542).

ATH: https://github.com/jenkinsci/blueocean-acceptance-test/pull/86

# Submitter checklist
- [ ] Link to JIRA ticket in description, if appropriate.
- [ ] Change is code complete and matches issue description
- [ ] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.
- [ ] Ran Acceptance Test Harness against PR changes.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

@reviewbybees

### CODE REVIEW

File: blueocean-dashboard/src/test/js/testResult-spec.js [hunk 1]
```diff
@@ -9,117 +9,143 @@ import { i18nTranslator } from '@jenkins-cd/blueocean-core-js';
...
+            "duration": 0.008, "empty": false, "failCount": 1, "passCount": 3, "skipCount": 0, "suites": [
+                {
+                    "duration": 0, "id": null, "name": "failure.TestThisWontFail", "stderr": null, "stdout": null, "timestamp": null, "cases": [
+                        { "age": 0, "className": "failure.TestThisWontFail", "duration": 0, "errorDetails": 'null', "errorStackTrace": 'aa', "failedSince": 0, "name": "aPassingTest2", "skipped": false, "skippedMessage": null, "status": "FAILED", "stderr": null, "stdout": null },
+                    ],
+                }]
+        };
+
+        let wrapper =mount(<TestResults t={t} testResults={successWithFixed} />);
+        const clickable = wrapper.find('.new-failure-block');
```
Comment (imeredith) [inline blueocean-dashboard/src/test/js/testResult-spec.js]: This is what is failing

Comment (tfennelly) [inline blueocean-dashboard/src/test/js/testResult-spec.js]: In what way is it failing?

**Comment by michaelneale** [comment]
paging @cliffmeyers - any ideas?

**Comment by imeredith** [comment]
Cliff helped me sort this out. PR fixes incoming shortly

**Comment by michaelneale** [comment]
ATH running: https://ci.blueocean.io/job/ATH-Jenkinsfile/job/JENKINS-39542-trim-null/2/

**Comment by michaelneale** [comment]
:bee: this runs nicely - good one @imeredith

**Pull request merged**